### ISSUE TITLE
Final Peer Review ls769

### ISSUE THREAD

**Issue by LorenzoSdV** [caperstar/ORIE5741#14]
This project is about how loan companies should accept or reject applicants for credit card or loans. The objective is to predict whether or not an applicant applying for a loan should be approved. They use the HMDA data of New York provided by FFIEC.

I like how you removed outliers in different ways according to a feature's distribution. I like that you kept a baseline model in your analysis (logistic regression) even though it was not great, and you improved it with PCA. I like that you provided a confusion matrix for each, not just your best model to show transparency. Since you used GBDT, it would be interesting to know which features are most important to GBDT vs which features are important according to ControlBurn. Instead of oversampling to have the same amount of data of each label, you should compare how your models do with cutting enough of the bigger size of data with one label to equal the amount of the smaller size of data with the other label. I would suggest mentioning why you chose the to display those 4 error metrics and how do they compare with each other across each model. I would clerkly mention how you split your data (training, validation, and test sets), and how did you do hyperparametrization if at all?
